lib/ain-grpc/src/errors.rs

```diff
@@ -2,33 +2,57 @@ use ain_evm::EVMError;
 use jsonrpsee::core::Error;
 
 pub enum RPCError {
-    EvmCall(EVMError),
+    AccountError,
+    BlockNotFound,
+    Error(Box<dyn std::error::Error>),
+    EvmError(EVMError),
+    FromBlockGreaterThanToBlock,
+    GasCapTooLow(u64),
     InsufficientFunds,
-    ValueOverflow,
+    InvalidBlockInput,
+    InvalidDataInput,
+    InvalidLogFilter,
     InvalidGasPrice,
+    InvalidTransactionMessage,
     InvalidTransactionType,
-    InvalidDataInput,
+    NonceCacheError,
+    StateRootNotFound,
     TxExecutionFailed,
-    GasCapTooLow(u64),
+    ValueOverflow,
 }
 
 impl From<RPCError> for Error {
     fn from(e: RPCError) -> Self {
         match e {
-            RPCError::EvmCall(e) => Error::Custom(format!("error calling EVM : {e:?}")),
+            RPCError::AccountError => to_custom_err("error getting account"),
+            RPCError::BlockNotFound => to_custom_err("block not found"),
```

Comment:
Any reason to change error message here ? `header not found` keeps us in sync with `geth`

Reply:
No reason, i'll switch it back